exporter/templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.81.xml

```diff
@@ -0,0 +1,21 @@
+<entry>
+  <encounter classCode="ENC" moodCode="EVN">
+    <!-- Transfer From -->
+    <templateId root="2.16.840.1.113883.10.20.24.3.81" extension="2014-12-01"/>
+    <id root="1.3.6.1.4.1.115" extension="{{.EntrySection.Entry.ObjectIdentifier.ID}}"/>
```

Comment:
In HDS this identifier is a function of the entry and the transferFrom within it, so I think we may want to generate it from more than just the entry ID, so that it's different from the ID in the parent template